### PR TITLE
Update BD generalist job ad

### PR DESCRIPTION
**Pull request by moyni**
@nqureshi all cool?

### CODE REVIEW

File: app/pages/about/_bd_generalist.html [hunk 1]
```diff
@@ -1,61 +1,83 @@
...
   <li>
-    Extremely smart with strong problem-solving and analytical skills
+    You execute on the details while keeping the big picture in mind
   </li>
   <li>
-    Relentless work ethic and drive
+    You're adaptive and keen to pick up new skills
+  </li>
+  <li>
+    You're driven and self-motivating
```
Comment (nqureshi) [inline app/pages/about/_bd_generalist.html]: *motivated?

File: app/pages/about/_bd_generalist.html [hunk 2]
```diff
@@ -1,61 +1,83 @@
...
 
 <p class="para">
-  Bonus points for applicants with a passion for technology startups.
-</p>
-
-<p class="para">
-  Industry competitive salary. This role would ideally suit someone with 2-5 
-  years' professional experience at a leading consultancy or technology firm. Exceptional candidates with alternative backgrounds and a proven track record should also apply.
+  Our current team come from a variety of backgrounds: mechanical engineering, 
+  economics, consulting, finance, entrepreneurships &amp; startups. We welcome
```
Comment (nqureshi) [inline app/pages/about/_bd_generalist.html]: *entrepreneurship

**Comment by nqureshi** [comment]
@moyni couple of minor (possible) typos and then :+1:

**Comment by moyni** [comment]
@nqureshi fixed.

**Pull request merged**